acceptance/configure_bosh_test.go

```diff
@@ -570,6 +575,11 @@ var _ = Describe("configure-bosh command", func() {
 			Expect(Forms[0].Get("iaas_configuration[bosh_vm_folder]")).To(Equal("some-vm-folder"))
 			Expect(Forms[0].Get("iaas_configuration[bosh_template_folder]")).To(Equal("some-template-folder"))
 			Expect(Forms[0].Get("iaas_configuration[bosh_disk_path]")).To(Equal("some-disk-path"))
+			Expect(Forms[0].Get("iaas_configuration[nsx_networking_enabled]")).To(Equal("true"))
```

Comment:
Hi @christianang. We would like to see the NXS configuration tested in a separate context. We think it will make it more clear that those are optional parameters. 

Thanks so much

Josh && @kkallday